src/librustc_ast_lowering/expr.rs

```diff
@@ -202,7 +202,12 @@ impl<'hir> LoweringContext<'_, 'hir> {
             ExprKind::Mac(_) => panic!("Shouldn't exist here"),
         };
 
-        hir::Expr { hir_id: self.lower_node_id(e.id), kind, span: e.span, attrs: e.attrs.clone() }
+        hir::Expr {
+            hir_id: self.lower_node_id(e.id),
+            kind,
+            span: e.span,
+            attrs: e.attrs.iter().map(|a| self.lower_attr(a)).collect::<Vec<_>>().into(),
```

Comment:
Shouldn't this be using `attrs: self.lower_attrs(&e.attrs),`?

cc @Zoxc

Reply:
No, this uses `ThinVec`.

Reply:
Ah.